### PR TITLE
[SPARK-21055][SQL][FOLLOW-UP] replace grouping__id with grouping_id()

### PR DESCRIPTION
**Pull request by gatorsmile**
## What changes were proposed in this pull request?
Simplifies the test cases that were added in the PR https://github.com/apache/spark/pull/18270.

## How was this patch tested?
N/A

### CODE REVIEW

**Comment by gatorsmile** [comment]
cc @jiangxb1987 @cloud-fan

**Comment by gatorsmile** [comment]
retest this please

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala [hunk 1]
```diff
@@ -1376,223 +1376,125 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   }
 
   test("SPARK-8976 Wrong Result for Rollup #1") {
```
Comment (dongjoon-hyun) [inline sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala]: Nit, do we need to keep `SPARK-21055` in the test case name here, too?

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

File: sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala (hunk 1 above)
Comment (gatorsmile) [reply sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala]: That is fine. We do not have such a rule.

**Comment by gatorsmile** [comment]
Thanks! Merged to master.

**Pull request closed without merging**